src/components/views/messages/MPollBody.tsx

```diff
@@ -169,6 +172,43 @@ export function isPollEnded(
     return authorisedRelations.length > 0;
 }
 
+export function pollAlreadyHasVotes(mxEvent: MatrixEvent, getRelationsForEvent?: GetRelationsForEvent): boolean {
+    if (!getRelationsForEvent) return false;
+
+    const voteRelations = createVoteRelations(getRelationsForEvent, mxEvent.getId());
+    return voteRelations.getRelations().length > 0;
+}
+
+export function launchPollEditor(mxEvent: MatrixEvent, getRelationsForEvent?: GetRelationsForEvent): void {
```

Comment:
I don't see launchPollEditor being deleted from MessageActionBar is the diff - did I miss it, or was it deliberate?